pubspec.yaml

```diff
@@ -45,7 +45,7 @@ dependencies:
   znn_sdk_dart:
     git:
       url: https://github.com/alienc0der/znn_sdk_dart.git
```

Comment:
Can you please update the url to /zenon-network?
I prefer ref `master` but we can use a hash too.